spec/factories/course_condition_assessments.rb

```diff
@@ -3,7 +3,7 @@
           class: Course::Condition::Assessment.name, aliases: [:assessment_condition] do
     course
     assessment
-    association :conditional, factory: :course_achievement
+    association :conditional, factory: :assessment
```

Comment:
after declaring the traits like this
```
trait :achievement_conditional do
      association :conditional, factory: :achievement
    end

    trait :assessment_conditional do
      association :conditional, factory: :assessment
    end``
How to use them inside the factory? When removing this line, FactoryGirl throws an error because the conditional id is not defined.